### PR TITLE
Update what-the-sensitive-information-types-look-for.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected credit card format. After testing.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1268

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/what-the-sensitive-information-types-look-for.md [hunk 1]
```diff
@@ -1973,7 +1973,7 @@ A DLP policy is 75% confident that it's detected this type of sensitive informat
 
 ### Format
 
-16 digits which can be formatted or unformatted (dddddddddddddddd) and must pass the Luhn test.
+14 to 16 digits which can be formatted or unformatted (dddddddddddddddd) and must pass the Luhn test.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/what-the-sensitive-information-types-look-for.md]:
```suggestion
14 to 16 digits which can be formatted or unformatted (dddddddddddddddd) and which must pass the Luhn test.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #1268.

Thank you.

cc: @kenwith

**Pull request merged**